web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateAlert.tsx

```diff
@@ -0,0 +1,72 @@
+import { Alert } from "@/components/njwds-extended/Alert";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { getProfileConfig } from "@/lib/domain-logic/getProfileConfig";
+import { ProfileContentField } from "@/lib/types/types";
+import { ReactElement } from "react";
+
+interface Props {
+  fieldErrors: string[];
+}
+
+export const AnytimeActionTaxClearanceCertificateAlert = (props: Props): ReactElement | null => {
+  const { Config } = useConfig();
+
+  const fieldErrorsOrder = [
+    "requestingAgencyId",
+    "businessName",
+    "entityId",
+    "addressLine1",
+    "addressLine2",
+    "addressCity",
+    "addressState",
+    "addressZipCode",
+    "taxId",
+    "taxPin",
+  ];
+  const fieldErrors = fieldErrorsOrder.filter((item) => props.fieldErrors.includes(item));
+
+  const getLabel = (field: string): string => {
+    // Necessary because Config.formation.fields defaults to type 'any'
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-expect-error
+    const formationLabel = Config.formation.fields[field]?.label;
+
+    let requestingAgencyLabel = "";
+
+    if (field === "requestingAgencyId") {
+      requestingAgencyLabel = Config.taxClearanceCertificateStep2.requestingAgencyLabel;
+    }
+
+    // TODO: Should this be referencing the tax clearance fields?
```

Comment:
Purposefully left. I'm not sure the answer yet.

Reply:
The comment you left is an interesting one. I don’t think it makes sense to reference the tax clearance field names directly. For example, if you look at line 37, the field name is requestingAgencyId, but that’s not the label we want to display.